sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -281,10 +288,29 @@ public PersonDirectorySteps(
           webDriverHelpers.selectFromCombobox(PRESENT_CONDITION_FILTER_COMBOBOX, "Alive");
         });
 
+    When(
+        "I check if Date of dead for specified case is correct",
```

Comment:
please specify which case is about

Reply:
that is generic, it depends which case you choose:

    And I click on first person in person directory
    And I check if Date of dead for specified case is correct